build.gradle

```
@@ -41,6 +41,10 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-oauth2-client'
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.springframework.boot:spring-boot-starter-websocket'
+    implementation 'org.webjars:sockjs-client:1.1.2'
```

```suggestion
    implementation 'org.webjars:sockjs-client:1.1.2'
```

이거 클라이언트에게 내려주는 용도인가요?
맞다면 클라이언트에서 따로 패키지로 사용하니 제거해도될것같아요.